service/controller/app/resource/releasemigration/resource.go

```diff
@@ -181,7 +181,7 @@ func (r *Resource) ensureReleasesMigrated(ctx context.Context, k8sClient k8sclie
 	}
 
 	n := func(err error, t time.Duration) {
-		r.logger.Log("level", "debug", "message", "failed to migrate all helm v2 releases")
+		r.logger.Log("level", "debug", "message", "migration not complete")
```

Comment:
This retry is waiting for the migration job to complete. So I think this is clearer as nothing is failed. WDYT?

Reply:
I agreed, much better to understand.